Core/CoreUITests/CoreUITestCase.swift

```diff
@@ -303,6 +303,13 @@ open class CoreUITestCase: XCTestCase {
         mockURL(url, data: data, response: response, error: error, noCallback: noCallback)
     }
 
+    open func mockGraphQL(_ json: Any) {
+        let data = try! JSONSerialization.data(withJSONObject: json, options: [])
+        let api = URLSessionAPI()
+        let url = URL(string: "/api/graphql", relativeTo: api.baseURL)!
+        mockURL(url, data: data, response: nil, error: nil, noCallback: false)
+    }
```

Comment:
Some way to mock more than 1 graphQL query at a time would probably be good, but that's a future bridge to burn.